Dockerfile

```diff
@@ -26,11 +26,14 @@ ENTRYPOINT [ "/tini", "--", "./bin/application" ]
 CMD [ "serve" ]
 
 ADD ./package.json ./
+ADD ./yarn.lock ./
 RUN apk --update add curl git && \
   curl -Ls "https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz" | tar xz -C / && \
-  npm install --unsafe-perm --loglevel error && \
+  npm install yarn --global && \
```

Comment:
just curious, if Docker still requires npm to work, what's the advantage? Just faster installs?